### PR TITLE
Update flake + switch to dream2nix

### PR DESCRIPTION
**Pull request by tinybeachthor**
- Update flake schema
- Update source repo to new url
- Update alpha75 -> alpha80
- Update flake inputs
- Migrate node2nix -> dream2nix

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -51,31 +54,22 @@
...
         inherit system;
         overlays = [
-          self.overlay
+          self.overlays.default
           (final: prev: { update-nixified-deps = final.callPackage update-nixified-deps {}; })
         ];
       });
 
       # It's a shame this is necessary. Alas. Run this after updating the
       # castopod-host-src input: `nix run .#update-nixified-deps`
```
Comment (DieracDelta) [inline flake.nix:63]: Is this still true?

**Review by DieracDelta — APPROVED** [review]
LGTM, nice work! Might be worthwhile updating the README since it refers to a bunch of the no longer existing node2nix files.

**Comment by tomberek** [comment]
> LGTM, nice work! Might be worthwhile updating the README since it refers to a bunch of the no longer existing node2nix files.

Looks good, but does require some text or explanation of how to use/upgrade the dream2nix locks.

**Comment by tinybeachthor** [comment]
@tomberek there are no dream2nix locks, the original repository has a `package-lock.json` file and dream2nix can use that. The update script is still needed for php dependencies (dream2nix does not support it yet).

@DieracDelta I have update the README updating instructions + added a short dream2nix explanation.

File: flake.nix (hunk 1 above)
Comment (tinybeachthor) [reply flake.nix:63]: Yes, this is still needed for php dependencies (dream2nix does not support php yet).

**Review by DieracDelta — APPROVED** [review]
LGTM!

**Pull request merged**